packages/bits/demo/src/components/demo/wizard-v2/wizard-restore-state/wizard-restore-state.example.component.ts

```diff
@@ -4,7 +4,7 @@ import { DialogService, NuiDialogRef, WizardStepV2Component, IWizardState } from
 import isEqual from "lodash/isEqual";
 import { Subject } from "rxjs";
 import { takeUntil } from "rxjs/operators";
-import { IWizardStepConfig } from "../wizard-dynamic/wizard-dynamic.example.component";
+import { IWizardStepV2Component } from "../wizard-dynamic/wizard-dynamic.example.component";
```

Comment:
Same comment as above. Can we duplicate this interface in here to make the example code self-contained (instead of importing it from another example).

Reply:
fixed